### PR TITLE
Refactor: refactoring Readme file

### PR DESCRIPTION
**Pull request by eapg**
* Reactoring `README` file to clarify steps to setup project.
* Removing jwt dependency from requirements.txt

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,20 +42,51 @@ Download and install the following tools:
...
+* In order to install library dependency run the following command : `pip install -r requirements.txt`
 * In order to update library dependency run the following command : `pip freeze > requirements.txt`
 * In order to set Environment variables file :
   * - Use `.env.local.example` file to create the different environment files: local, dev, test. 
   * - choose under which environment you are going to work using command: `SET ENV=local` for local, `SET ENV=test` for
       testing.
   * - Run the script that you want to run under chosen environment.
+
+### setup Data base:
+* create `POSTGRESQL` database with the name: `ea_restaurant`
```
Comment (reynoldmorel) [inline README.md:63]: You mean `Create`?

File: README.md [hunk 2]
```diff
@@ -42,20 +42,51 @@ Download and install the following tools:
...
+* Run migrations: 
+
+Executing `alembic upgrade head` in a clean database will execute all migrations
+in the orders that migrations were created. Like example image.
+
+![img.png](img.png)
+
+To know more about migrations refer to migrations section.
+
+* Create `MONGODB` database with name: `ea_restaurant` and create a collection with name:`order_status_histories`
```
Comment (reynoldmorel) [inline README.md:73]: You need to mention what tools you suggest to create a mongo db and also show the steps for the tool you use.

File: README.md (hunk 1 above)
Comment (reynoldmorel) [reply README.md:63]: You need to mention what tools you suggest to create a mongo db and also show the steps for the tool you use.

**Review by reynoldmorel — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (eapg) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (eapg) [reply README.md:80]: Done.

**Review by reynoldmorel — APPROVED** [review]

**Pull request merged**